app/common/data/types.py

```diff
@@ -36,3 +36,7 @@ class CollectionStatusEnum(enum.StrEnum):
     NOT_STARTED = "Not started"
     IN_PROGRESS = "In progress"
     COMPLETED = "Completed"
+
+
+class MetadataEventKey(enum.StrEnum):
+    FORM_RUNNER_FORM_COMPLETED = "Form completed"
```

Comment:
```suggestion
    FORM_RUNNER_FORM_MARKED_COMPLETE = "Form marked as complete"
    FORM_RUNNER_FORM_MARKED_INCOMPLETE = "Form marked as incomplete"
```

maybe?

Reply:
> To try out this method we're just using a very simple method of removing events
that are no longer relevant (if a user decides a form is no longer complete).

Ah I see your comment.

I don't love the idea of deleting audit-trail-esque events very much 👀